app/views/courses/TeacherCoursesView.coffee

```diff
@@ -48,91 +42,26 @@ module.exports = class TeacherCoursesView extends RootView
     @courses = new CocoCollection([], { url: "/db/course", model: Course})
     @supermodel.loadCollection(@courses, 'courses')
-    @classrooms = new CocoCollection([], { url: "/db/classroom", model: Classroom })
-    @classrooms.comparator = '_id'
-    @listenToOnce @classrooms, 'sync', @onceClassroomsSync
-    @supermodel.loadCollection(@classrooms, 'classrooms', {data: {ownerID: me.id}})
     @campaigns = new Campaigns()
     @supermodel.trackRequest @campaigns.fetchByType('course', { data: { project: 'levels,levelsUpdated' } })
-    @courseInstances = new CocoCollection([], { url: "/db/course_instance", model: CourseInstance })
-    @courseInstances.comparator = 'courseID'
-    @courseInstances.sliceWithMembers = -> return @filter (courseInstance) -> _.size(courseInstance.get('members')) and courseInstance.get('classroomID')
-    @supermodel.loadCollection(@courseInstances, 'course_instances', {data: {ownerID: me.id}})
-    @members = new CocoCollection([], { model: User })
-    @listenTo @members, 'sync', @render
     @
 
-  onceClassroomsSync: ->
-    for classroom in @classrooms.models
-      @members.fetch({
-        remove: false
-        url: "/db/classroom/#{classroom.id}/members"
-      })
-
-  onClickActivateLicensesButton: ->
-    modal = new ActivateLicensesModal({
-      users: @members
-    })
-    @openModalView(modal)
-    modal.once 'redeem-users', -> document.location.reload()
-    application.tracker?.trackEvent 'Courses teachers started enroll students', category: 'Courses'
-
-  onClickAddStudents: (e) ->
-    classroomID = $(e.target).data('classroom-id')
-    classroom = @classrooms.get(classroomID)
-    unless classroom
-      console.error 'No classroom ID found.'
-      return
-    modal = new InviteToClassroomModal({classroom: classroom})
-    @openModalView(modal)
-    application.tracker?.trackEvent 'Classroom started add students', category: 'Courses', classroomID: classroom.id
+  initialize: (options) ->
+    window.tracker?.trackEvent 'Classes Guides Loaded', category: 'Teachers', ['Mixpanel']
+    super(options)
 
-  onClickCreateNewClassButton: ->
-    return application.router.navigate('/teachers/signup', {trigger: true}) if me.get('anonymous')
-    modal = new ClassroomSettingsModal({})
-    @openModalView(modal)
-    @listenToOnce modal, 'hide', =>
-      # TODO: how to get new classroom from modal?
-      @classrooms.add(modal.classroom)
-      # TODO: will this definitely fire after modal saves new classroom?
-      @listenToOnce modal.classroom, 'sync', ->
-        @addFreeCourseInstances()
-        @render()
+  onClickGuideButton: (e) ->
+    courseID = $(e.currentTarget).data('course-id')
+    courseName = $(e.currentTarget).data('course-name')
+    eventAction = $(e.currentTarget).data('event-action')
+    window.tracker?.trackEvent eventAction, category: 'Teachers', courseID: courseID, courseName: courseName, ['Mixpanel']
+    window.location.href = $(e.currentTarget).data('href')
```

Comment:
Same here: Use normal `href`. This JS executes before the page change occurs anyway.